command/server.go

```diff
@@ -136,12 +135,10 @@ type ServerCommand struct {
 	flagDevSkipInit        bool
 	flagDevThreeNode       bool
 	flagDevFourCluster     bool
-	flagDevTransactional   bool
```

Comment:
We should probably add this back, to allow the dev server to use the transactional in-mem backend.

Reply:
As above, let's leave it enabled by default instead for dev mode.